core/api/type/type_proto/keymaster.proto

```diff
@@ -48,8 +49,9 @@ message SigningKey {
 message VerifyingKey {
   // KeyStatus defines a key status.
   enum KeyStatus {
-    ACTIVE = 0;      // indicates a key that is currently being used.
-    DEPRECATED = 1;  // indicates a key that is not in use.
+    UKNOWN = 0;
+    ACTIVE = 2;      // indicates a key that is currently being used.
```

Comment:
Should it be ACTIVE = 1, DEPRECATED = 3?
Or are you trying to keep numbers in sync with those in KeyStatus?
(If so, I wouldn't try)

Reply:
I'm not explicitly trying to keep them in sync, but it's not a bad idea..